lib/gems/pending/appliance_console.rb

```diff
@@ -597,7 +600,9 @@ module ApplianceConsole
 
       when I18n.t("advanced_settings.scap")
         say("#{selection}\n\n")
-        ApplianceConsole::Scap.new.lockdown
+        ApplianceConsole::Scap.new(
+          :rules_dir => File.expand_path("productization/appliance_console/config", RAILS_ROOT)
+        ).lockdown
```

Comment:
This feels weird that we enforce productization here, but I guess it will have to do.